config/samples/go_example_loadtest_with_pre_built_workers.yaml

```
@@ -0,0 +1,128 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: prebuilt-go-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: go
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+    - language: go
+      run:
+        image: ${prebuilt_image_prefix}/go:${prebuilt_image_tag}
+        command: ["/executable/worker"]
+
+
+  # Users can specify multiple clients. They are bound by the
+  # number of nodes.
+  clients:
+    - language: go
+      run:
+        image: ${prebuilt_image_prefix}/go:${prebuilt_image_tag}gcloud container clusters get-credentials benchmarks-dev1 \
+          --zone us-central1-b \
+          --project grpc-testing
```

It looks like some of your history got included. I think you meant this:

```suggestion
        image: ${prebuilt_image_prefix}/go:${prebuilt_image_tag}
```

---

Right! Thank you for pointing this out!